hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMDBCheckpointServlet.java

```diff
@@ -669,7 +669,9 @@ public BootstrapStateHandler.Lock lock()
       snapshotDeletingService.getBootstrapStateLock().lock();
 
       // Then wait for the double buffer to be flushed.
-      om.getOmRatisServer().getOmStateMachine().awaitDoubleBufferFlush();
+      if (om.isRatisEnabled()) {
+        om.getOmRatisServer().getOmStateMachine().awaitDoubleBufferFlush();
+      }
```

Comment:
`OzoneManager.awaitDoubleBufferFlush()` handles both cases (Ratis enabled/disabled).

```suggestion
      om.awaitDoubleBufferFlush();
```